src/main/java/model/Lotto.java

```diff
@@ -0,0 +1,56 @@
+package model;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.toList;
+
+public class Lotto {
+    private static final int LOTTO_SIZE = 6;
+    private static final String LOTTO_SIZE_MUST_BE_SIX = "로또 번호는 중복되지 않는 6개의 번호여야 합니다.";
+    public static final BigDecimal LOTTO_PRICE = new BigDecimal(1000);
```

Comment:
구글 코드 컨벤션에는 이런 내용이 안보이는데..... 
public 메서드를 private 메서드보다 더 위에 명시해주는 것이 좋습니다.

```suggestion
    public static final BigDecimal LOTTO_PRICE = new BigDecimal(1000);
    private static final int LOTTO_SIZE = 6;
    private static final String LOTTO_SIZE_MUST_BE_SIX = "로또 번호는 중복되지 않는 6개의 번호여야 합니다.";
```

Reply:
놓쳤던 부분인 것 같습니다. 반영하도록 하겠습니다.